tstester/config.json

```diff
@@ -1,39 +1,31 @@
 {
     "max_age": 86400,
-    "_comment": "max age in secs relative to current time (older observations are inaccessible)",
-
+    "_comment": "extra secs values to try with the AddNewObs test",
     "nstations": 3,
-    "_comment": "number of stations to generate",
-
     "bbox": {
         "min_lat": -60.5,
         "max_lat": 62.5,
         "min_lon": -10.5,
         "max_lon": 12.5
     },
-    "_comment": "bounding box for randomly generated station locations (no two stations will have the same location)",
-
     "params": {
         "min": 1,
         "max": 3
     },
-    "_comment": "minimum and maximum number of randomly generated params for a station",
-
     "time_res": {
         "60": 0.2,
         "600": 0.3,
         "3600": 0.5
     },
-    "_comment": "probability weights of time series resolutions (around 20% will have time res 60 secs, around 30% will have time res 600 secs, and so on)",
-
-    "extra_secs": [60, 600, 3600],
-    "_comment": "extra secs values to try with the AddNewObs test",
-
+    "extra_secs": [
+        60,
+        600,
+        3600
+    ],
```

Comment:
Keep all the comments by renaming them to _comment1, _comment2, etc.

Reply:
Well seen, I did not notice that the pre-commit hook removed some of the comments.

Reply:
It doesn't accept duplicates... :-)